framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -22,6 +22,7 @@
 import org.apache.drill.test.framework.TestCaseModeler.TestMatrix;
 import org.apache.drill.test.framework.TestVerifier.TestStatus;
 import org.apache.drill.test.framework.TestVerifier.VerificationException;
+import org.apache.drill.test.framework.TestVerifier.PlanVerificationException;
```

Comment:
Can we change the other to DataVerificationException?

Reply:
I kept VerificationException because this is "open-source" and I was wondering if someone might expect "Verification Failure" to still work.

Reply:
I don't think that should impact any consumer - if they rebase, they should get the latest changes. Previous forks should condinue to work. We could always simply keep the VerificationException type, but not use it anywhere in the code.

Reply:
Done.